### ISSUE TITLE
No data loaded when you pass a self-constructed model for trainer

### ISSUE THREAD

**Issue by visionshao** [visionshao/LLMisAllYouNeed#2]
When you create a new model for trainer, you should indicate the inputs for forward function. Like

forward(self, input_ids, attention_mask, labels, **kwargs):

The "input_ids", "attention_mask" are the output  variables of preprocess_function for data.map().

If the parameters of forward do not contain the output of preprocess_function, no data will be loaded due to the existing of remove_unused_column in trainer class.
